src/silo/database.test.cpp

```diff
@@ -1,11 +1,14 @@
 #include "silo/database.h"
 
 #include <gtest/gtest.h>
+#include <silo/query_engine/query_result.h>
```

Comment:
(minor thing) we tried to sort the includes in such a way that we have 4 blocks
 * Corresponding header file (for source files)
 *   System includes
 *   External includes
 *   Internal includes